### PR TITLE
[cdc-connector][sqlserver] sqlserver support use specific chunk column as a split key

### LINKED ISSUE #2971: [cdc-connector][sqlserver] sqlserver support use specific chunk column as a split key
### Search before asking

- [X] I searched in the [issues](https://github.com/ververica/flink-cdc-connectors/issues) and found nothing similar.

### Motivation

sqlserver support use specific chunk column as a split key.
sqlserver cdc support specific chunk column but it don't implement this function.

### Solution

I will  implement this function.

### Alternatives

_No response_

### Anything else?

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by gong**
### Prepare a Pull Request

-  Fixes #2971 

### Motivation

sqlserver support use specific chunk column as a split key

### Modifications

* Modify SqlServerUtils#getSplitColumn by adding a `chunkKeyColumn` param
* Add doc about chunk column
* Add ITcase UT about using chunk coulmn

### CODE REVIEW

**Comment by gong** [comment]
@leonardBang @GOODBOY008 PTAL

**Comment by gong** [comment]
@lvyanquan Hello, please help me to address UT fail about ValuesDataSourceITCase#testMultiSplitsSingleTable.

**Comment by gong** [comment]
@lvyanquan PTAL

**Review by lvyanquan — APPROVED** [review]
LGTM.

File: flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/main/java/com/ververica/cdc/connectors/sqlserver/source/utils/SqlServerUtils.java [hunk 1]
```diff
@@ -168,15 +170,27 @@ public static Column getSplitColumn(Table table) {
...
 
+        if (chunkKeyColumn != null) {
+            Optional<Column> targetPkColumn =
+                    primaryKeys.stream()
+                            .filter(col -> chunkKeyColumn.equals(col.name()))
+                            .findFirst();
+            if (targetPkColumn.isPresent()) {
+                return targetPkColumn.get();
+            }
+            throw new ValidationException(
```
Comment (leonardBang) [inline flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/main/java/com/ververica/cdc/connectors/sqlserver/source/utils/SqlServerUtils.java:181]: minor: Could we add a test to cover this branch? appreciated if you'd like to add for other connetors.

Comment (gong) [reply flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/main/java/com/ververica/cdc/connectors/sqlserver/source/utils/SqlServerUtils.java:181]: @leonardBang ok, I will add exception test.

**Review by leonardBang — APPROVED** [review]
Thanks @gong for the update, LGTM

**Pull request merged**